assets/css/edd-admin.css

```diff
@@ -2431,6 +2420,92 @@ body.download_page_edd-reports {
 	margin-bottom: 20px;
 }
 
+div#post-body-content {
```

Comment:
@LisaCee This will affect all screens EDD is loaded on, and used by core WordPress markup so you'll get things like:

<img width="806" alt="Screen Shot 2020-09-16 at 2 25 16 PM" src="https://user-images.githubusercontent.com/491124/93377026-77c4fc00-f828-11ea-99e2-bc17ddeeda07.png">

Instead you can add a class to the wrapping `<div>` here:

https://github.com/easydigitaldownloads/easy-digital-downloads/blob/release/3.0/includes/admin/reporting/reports.php#L2555

Maybe something like `.edd-reports-export` and then you can create the grid just on that class.